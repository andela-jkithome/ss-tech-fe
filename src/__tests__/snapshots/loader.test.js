import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../../components/loader';

const fetching = true;

it('Renders correctly when fetching', () => {
  const component = renderer.create(<Loading fetching={fetching} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it('Renders correctly when not fetching', () => {
  const component = renderer.create(<Loading fetching={false} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
