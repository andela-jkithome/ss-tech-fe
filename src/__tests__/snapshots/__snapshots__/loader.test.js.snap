// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly when fetching 1`] = `
<div
  className="ui centered grid container"
>
  <div
    className="column"
    columns={1}
  >
    <div
      className="row"
    >
      <div
        className="column"
      >
        <div
          className="ui active indicating progress"
          data-percent={100}
        >
          <div
            className="bar"
            style={
              Object {
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="column"
      >
        <div
          className="ui blue massive center aligned segment"
        >
          Fetching Categories...
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Renders correctly when not fetching 1`] = `
<div
  className="ui centered grid container"
>
  <div
    className="column"
    columns={1}
  >
    <div
      className="row"
    >
      <div
        className="column"
      >
        <div
          className="ui raised tertiary center aligned segment"
        >
          Click on any one of the categories listed to display a 
            random joke from the category.
        </div>
      </div>
    </div>
  </div>
</div>
`;
